rule/early_return.go

```diff
@@ -62,6 +62,11 @@ func (e *EarlyReturnRule) checkIfElse(chain ifelse.Chain) (string, bool) {
 		return "", false
 	}
 
+	if chain.If.Branch().Deviates() && chain.If.Branch().IsShort() {
```

Comment:
How this condition asserts that it is a _single statement **error check**_ ?

Reply:
I'm afraid this is not possible due to the fact that `early-return` is untyped.

Reply:
Thus the comment in the next line
`// Don't suggest refactor for single-statement error checks (guard clauses)` can be misleading.

Should we said `// Don't suggest refactor for single-statement ` instead ?